src/uport.js

```diff
@@ -1,24 +1,30 @@
-import MsgServer from './msgServer'
-import { Registry } from 'uport-persona'
+import TopicFactory from './topicFactory'
+// import UportJs from 'uport'
+// import UportLite from 'uport-lite'
 import MobileDetect from 'mobile-detect'
 import ContractFactory from './contract'
 import UportWeb3 from './uportWeb3'
-
-
-//TODO Only our default now (maybe), not customizable, or minimally
+import { QRUtil } from './util/qrdisplay'
+// TODO Only our default now (maybe), not customizable, or minimally
 
 import { decodeToken } from 'jsontokens'
 
-const CHASQUI_URL = 'https://chasqui.uport.me/api/v1/topic/'
 // these are consensysnet constants, replace with mainnet before release!
 const INFURA_ROPSTEN = 'https://ropsten.infura.io'
-const UPORT_REGISTRY_ADDRESS = '0xb9C1598e24650437a3055F7f66AC1820c419a679'
```

Comment:
yeah if clear in the readme, not needed here. I've just spent a lot of time looking for defaults by only reading code unfortunately